knowledge-base/signing-a-document-with-digital-signature.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Signing a document with a digital signature
+description: Signing a document with a digital signature
+type: how-to
+page_title: Signing a document with a digital signature
+slug: signing-a-document-with-digital-signature
+position: 0
+tags: pdf, sign, digital, signature, HSM, USB, certificate, X509Certificate2
+res_type: kb
+---
+
+<table>
+<thead>
+	<tr>
+		<th>Product Version</th>
+		<th>Product</th>
+		<th>Author</th>
+	</tr>
+</thead>
+<tbody>
+	<tr>
+		<td>2020.1.310</td>
+		<td>RadPdfProcessing</td>
+		<td><a href="https://www.telerik.com/blogs/author/martin-velikov">Martin Velikov</a></td>
+	</tr>
+</tbody>
+</table>
```

Comment:
The other KBs have a different structure here.

Reply:
Both are valid and produce the same result.